manifests/j/Jabra/Direct/5.7.45144/Jabra.Direct.installer.yaml

```diff
@@ -0,0 +1,18 @@
+# Automatically updated by the winget bot at 2021/Aug/18
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+PackageIdentifier: Jabra.Direct
+PackageVersion: 5.7.45144
+MinimumOSVersion: 7.0.0.0
+InstallerType: exe
+InstallerSwitches:
+  Silent: /Install /Quiet /Norestart
+  SilentWithProgress: /Install /Passive /quiet
+  Interactive: /Install
+ProductCode: '{2183fe12-ffa1-4abe-a5a2-acffcee16462}'

```

Comment:
```suggestion
ProductCode: '{0f616985-7a39-4819-a042-7aba49da0ae7}'
```

Reply:
Even though there's two Product Code, if you delete `{0f616985-7a39-4819-a042-7aba49da0ae7}` in the registry, it disappears in Control Panel, so I chose this one.

Reply:
The ProductCode is probably only automatically updated for MSI's which is not needed in this case so i removed it in https://github.com/microsoft/winget-pkgs/pull/25471